### PR TITLE
add watermark param in docs

### PR DESCRIPTION
**Pull request by ArtyomHov**

### CODE REVIEW

**Comment by narekhovhannisyan** [comment]
@ArtyomHov please look up how commit messages are written in this repository, how release commit message looks like, don't add multiple files in one commit.

**Comment by narekhovhannisyan** [comment]
and please update version to 0.3.8 and commit package, package-lock files with one commit with message like this -> ✨ release 0.3.8

File: README.md [hunk 1]
```diff
@@ -271,7 +271,8 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   projectId: 5000658,
-  quality: 1080
+  quality: 360,
+  watermark: 'watermarkURL'
```
Comment (emarukyan) [inline README.md]: watermark: 'https://somedomain.com/watermark.png' // optional argument

File: README.md [hunk 2]
```diff
@@ -271,7 +271,8 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   projectId: 5000658,
-  quality: 1080
+  quality: 360,
+  watermark: 'https://somedomain.com/watermark.png' // optional argument
```
Comment (narekhovhannisyan) [inline README.md]: in the whole repository, it's a convention to use example.com for a mock value of the url.

File: README.md [hunk 3]
```diff
@@ -216,7 +216,7 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   projectId: 5000658,
-  quality: 360
+  quality: 360 //optional argument
```
Comment (narekhovhannisyan) [inline README.md]: in commit message why you do not mention which example is updated? commit message must describe the changes it's made. And please don't use commit reverting or similar things.

Comment (narekhovhannisyan) [reply README.md]: and please use full stops at the end of the commit message sentence :)

Comment (narekhovhannisyan) [reply README.md]: you can create a new branch, and do some beautiful commits there, and open pull request :)

**Pull request closed without merging**